src/include/btree_inline.h

```diff
@@ -1612,6 +1612,21 @@ __wt_page_del_visible_all(WT_SESSION_IMPL *session, WT_PAGE_DELETED *page_del, b
     if (page_del == NULL)
         return (true);
 
+    /*
+     * committed flag indicaes whether truncate transaction is committed or not.
```

Comment:
Can you refactor this, to have a new `page_del_visible_int` that does all the work shared between `wt_page_del_visible_all` and `wt_page_del_visible`?

Also - this comment could use some spelling and grammar review. Could you take a pass of that?